### PR TITLE
 change tensorflow runtime to tf v1.13.1 with mkl

### PR DESCRIPTION
**Pull request by yangw1234**

### CODE REVIEW

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/net/TFNet.scala [hunk 1]
```diff
@@ -578,6 +578,89 @@ class TFNet(private val graphDef: TFGraphHolder,
 
 object TFNet {
 
+  val iomp5name = "libiomp5.so"
```
Comment (jason-dai) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/net/TFNet.scala]: shall we move this to core?

Comment (yangw1234) [reply zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/net/TFNet.scala]: moved to core

File: pyzoo/dev/prepare_env.sh [hunk 2]
```diff
@@ -37,3 +37,8 @@ echo "ANALYTICS_ZOO_CLASSPATH": $ANALYTICS_ZOO_CLASSPATH
 
 export BIGDL_CLASSPATH=$ANALYTICS_ZOO_CLASSPATH
 echo "BIGDL_CLASSPATH": $BIGDL_CLASSPATH
+
+export KMP_BLOCKTIME=0
```
Comment (yangw1234) [inline pyzoo/dev/prepare_env.sh]: tried to use jni to call setenv in c++ from java side, but it does not work, not sure about why

Comment (yangw1234) [reply pyzoo/dev/prepare_env.sh]: anywhere else I should export those environment variables, like in pip install mode? @hkvision

Comment (hkvision) [reply pyzoo/dev/prepare_env.sh]: can we just export these in __init__ of zoo?

Comment (yangw1234) [reply pyzoo/dev/prepare_env.sh]: they need to be set before jvm are started

Comment (jason-dai) [reply pyzoo/dev/prepare_env.sh]: where is this script used? does it work in the cluster? how is this handled in BigDL MKLDNN support?

Comment (yangw1234) [reply pyzoo/dev/prepare_env.sh]: the script is used in python testing
In BigDL, it is handled by calling a few native methods like omp_set_num_threads, but in our case tensorflow will override that

File: zoo/src/main/resources/spark-analytics-zoo.conf [hunk 3]
```diff
@@ -31,4 +31,8 @@ spark.shuffle.reduceLocality.enabled                false
 spark.shuffle.blockTransferService                  nio
 spark.scheduler.minRegisteredResourcesRatio         1.0
 spark.speculation                                   false
-spark.serializer                                    org.apache.spark.serializer.JavaSerializer
\ No newline at end of file
+spark.serializer                                    org.apache.spark.serializer.JavaSerializer
+spark.executorEnv.KMP_BLOCKTIME                     0
+spark.executorEnv.KMP_AFFINITY                      granularity=fine,verbose,compact,1,0
+spark.executorEnv.KMP_SETTINGS                      1
+spark.executorEnv.OMP_NUM_THREADS                   1
```
Comment (yangw1234) [inline zoo/src/main/resources/spark-analytics-zoo.conf]: for cluster to work @jason-dai

File: pyzoo/dev/prepare_env.sh (hunk 2 above)
Comment (yangw1234) [reply pyzoo/dev/prepare_env.sh]: in cluster, the environment varible is exported using analytics-zoo-spark.conf

**Comment by yangw1234** [comment]
jenkins pass http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/1942/

**Pull request merged**